app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
 
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
```

Comment:
The file is located in AntennaPod's own directory, so the permission is not needed. What I meant with "grant read permission" is that AntennaPod should allow the other apps to read the file even if the other apps do not have the storage permission. You can see how this works by looking at: https://github.com/AntennaPod/AntennaPod/blob/72f7713fff9468c4dbe1b1fc57e733fba021eb80/core/src/main/java/de/danoeh/antennapod/core/util/ShareUtils.java#L75-L83